testscript/testscript.go

```diff
@@ -150,6 +150,10 @@ func RunT(t T, p Params) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	testTempDir, err = filepath.EvalSymlinks(testTempDir)
```

Comment:
Please add a comment explaining why this is needed.

Reply:
Struggled a bit with how to explain, but now there is a comment there at least, feel free to suggest improvements :)